# TODO other message types below
float time_imu
float time_gps
float northing
float easting
float yaw
float mag_x
float mag_y
